cmd/helm/status.go

```diff
@@ -74,6 +74,7 @@ func newStatusCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 
 type statusPrinter struct {
 	release *release.Release
+	debug   bool
```

Comment:
is this used?

Reply:
Yep, it is passed into the print command for `fullInfo`